packages/hardhat-core/test/internal/hardhat-network/provider/modules/debug.ts

```diff
@@ -353,14 +352,12 @@ describe.skip("Debug module", function () {
     });
 
     // see https://github.com/NomicFoundation/hardhat/issues/3519
-    it.skip("Should return the right values for a successful tx", async function () {
+    it("Should return the right values for a successful tx", async function () {
```

Comment:
Please note that this test was disabled before, but I assume due to a different error than what I’m seeing now.

The test fails with this error in `rethnet` mode:

`Error: Post-merge transaction is missing prevrandao`

And it fails due to a timeout in `ethereumjs` mode.

Reply:
The mentioned failure is resolved in https://github.com/NomicFoundation/hardhat/pull/4398/commits/60179e7b673d5f0c951ff9ec794ba786a8fef134

Now `rethnet` mode also has a timeout